tests/reference/asr-kwargs_02_error-7c91f8f.stderr

```diff
@@ -1,5 +1,5 @@
-semantic error: func02() got multiple values for argument 'b'
-  --> tests/errors/kwargs_02_error.py:13:5
-   |
-13 |     func02(arg3, arg4, c=arg5, b=arg4)
-   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
+semantic error: Type mismatch in assignment, the types must be compatible
+ --> tests/errors/kwargs_02_error.py:9:5
+  |
+9 |     arg3 = 3.0
+  |     ^^^^   ^^^ type mismatch ('f32' and 'f64')
```

Comment:
This error message has changed.

Reply:
Fixed in [b5b4c82](https://github.com/lcompilers/lpython/pull/1271/commits/b5b4c82342721669d793585c84d3c87b922f53ae)